retroshare-gui/src/util/MRichTextEdit.h

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
There's a license problem here. retroshare-gui is AGPLv3+. Depending on where you found this file, it may be possible to re-license it with AGPLv3+ as well,

Reply:
from here 
https://github.com/Anchakor/MRichTextEditor

Reply:
@csoler what now?

Reply:
Can you ask the author if you can relicense this file to AGPLv3+ ? If not, we cannot add this file. If it comes from Qt examples, it shouldn't be a problem if I remember correctly.